src/stores/period.store.spec.ts

```diff
@@ -32,12 +33,32 @@ describe('OrderStore store', () => {
   })
 
   it('should init with trades', async (done) => {
-    // @todo(notVitaliy): Add this test
+    const now = new Date().getTime();
```

Comment:
This test also tests the `newPeriod` method. I'm going to merge this and fix it up but in the future try to limit your test to only testing 1 method unless that method was already tested in an above test. e.g. In a test above this one, `addTrade` was already tested so it' fine if this test uses it but `newPeriod` wasn't yet so try to avoid it being invoked.